crates/proof-of-sql/src/sql/proof/verifiable_query_result.rs

```diff
@@ -88,7 +88,11 @@ impl<CP: CommitmentEvaluationProof> VerifiableQueryResult<CP> {
         // have been rejected at the parsing stage.
 
         // handle the empty case
-        if expr.is_empty(accessor) {
+        let table_refs = expr.get_table_references();
+        if table_refs
+            .iter()
```

Comment:
NIT:
You could maybe consume `table_refs`.
```suggestion
            .into_iter()
```